testsuites/CBLTester/CBL_Functional_tests/TestSetup_FunctionalTests/test_replication_eventing.py

```diff
@@ -0,0 +1,511 @@
+import pytest
+import random
+
+
+from keywords.MobileRestClient import MobileRestClient
+from keywords.utils import log_info
+from keywords.utils import get_event_changes
+from CBLClient.Database import Database
+from CBLClient.Replication import Replication
+from libraries.testkit import cluster
+
+
+@pytest.fixture(scope="function")
+def setup_teardown_test(params_from_base_test_setup):
```

Comment:
forgot to ask about this. This teardown is not used anywhere,so remove this .

Reply:
Removed